syftlib/src/main/java/org/openmined/syft/processes/SyftJob.kt

```diff
@@ -40,10 +41,12 @@ class SyftJob(
     private lateinit var requestKey: String
 
     //todo need to filled based on the destination directory defined by syft configuration class
-    private val destinationDir = ""
-    private val modelFileLocation = "$destinationDir/model/$modelName"
+    private val destinationDir = "/data/data/org.openmined.syft.demo/files"
```

Comment:
Maybe this can provided by the client since it depends on it. We don't have any guarantee that all phone models will have the same directory organisation.
We can leave it by now while we decide how to pass it (as you said, probably in some configuration class).

Reply:
Yeah it won't remain here. It will come from configuration class.